src/styles/blocks/_products.scss

```diff
@@ -0,0 +1,19 @@
+.products {
+  margin: 0 auto 73px;
+  padding-top: 50px;
+  max-width: $catalog-width;
+  // max-width: 1024px;
```

Comment:
remove comments

Reply:
Thank you for your review! Code was changed according to your comments